generate_markdown_from_doxygen_xml.py

```diff
@@ -0,0 +1,1008 @@
+"""
```

Comment:
Shebang missing, please add: `#!/usr/bin/env python`

Reply:
Done

Reply:
I don't see it.

Reply:
Will add. Sorry - somehow got removed.